requirements/ci.txt

```diff
@@ -132,7 +132,7 @@ click-repl==0.2.0
     #   celery
 commonmark==0.9.1
     # via recommonmark
-coverage==4.5.4
+coverage==7.1.0
```

Comment:
this was an attempt for the coverage config being picked up by codecov, but it doesn't seem to do something like that.

It appears that codecov at the moment includes the test code itself in the coverage measuring/reporting and disabling that will require a config file. Created #2665 for that